Source/WebCore/dom/DocumentMarkerController.cpp

```diff
@@ -380,6 +391,10 @@ void DocumentMarkerController::copyMarkers(Node& source, OffsetRange range, Node
 
 void DocumentMarkerController::removeMarkers(Node& node, OffsetRange range, OptionSet<DocumentMarker::Type> types, const Function<FilterMarkerResult(const DocumentMarker&)>& filter, RemovePartiallyOverlappingMarker overlapRule)
 {
+#if ENABLE(UNIFIED_TEXT_REPLACEMENT)
+    if (types.contains(DocumentMarker::Type::UnifiedTextReplacement))
+        m_unifiedTextReplacementAnimationTimer.stop();
+#endif
```

Comment:
I don't think this is right, because this method could only remove the markers on a single node, and not across the entire document.

`DocumentMarkerController::removeMarkers(OptionSet<DocumentMarker::Type> types, const Function<FilterMarkerResult(const RenderedDocumentMarker&)>& filter)` is probably a better place for this.